importer/urls.py

```diff
@@ -0,0 +1,12 @@
+"""
+URLs for the importer app.
+"""
+from django.conf.urls import url, patterns
```

Comment:
Patterns is no longer recommended: https://docs.djangoproject.com/en/1.8/releases/1.8/#django-conf-urls-patterns